todo/views.py

```diff
@@ -0,0 +1,71 @@
+from django.urls import reverse_lazy
+from django.views.generic import ListView, CreateView, UpdateView, DeleteView
+from django.views.generic.base import View
+from django.shortcuts import redirect
+from django.shortcuts import get_object_or_404
+from .models import Task, Tag
+from .forms import TaskForm, TagForm
```

Comment:
Add blank line between different import groups

use absolute import instead of relative